orttraining/orttraining/training_ops/cuda/optimizer/lamb.cu

```diff
@@ -440,9 +443,15 @@ INSTANTIATE_LAMB_MULTI_TENSOR_UPDATE_FUNCTOR(double, double, double, half)
 INSTANTIATE_LAMB_MULTI_TENSOR_UPDATE_FUNCTOR(half, float, half, half)
 INSTANTIATE_LAMB_MULTI_TENSOR_UPDATE_FUNCTOR(float, float, half, half)
 
+// w_buffer[i], d_buffer[i] is used to store the squared sum of all elements processed by the i-th block.
+// sync_range_and_lock[t] contains (start block, number of blocks, locking counter) for tensor t
```

Comment:
>(start block, number of blocks, locking counter) [](start = 35, length = 48)

could it just be a struct? then the fields can have names

Reply:
Yeah, good point.

Reply:
OK, should read easier now.